src/data/pokemon/base_stats.h

```diff
@@ -1,6 +1,37 @@
-// Maximum value for a female pokemon is 254 (MON_FEMALE) which is 100% female.
-// 255 (MON_GENDERLESS) is reserved for genderless pokemon.
+// Maximum value for a female Pok�mon is 254 (MON_FEMALE) which is 100% female.
+// 255 (MON_GENDERLESS) is reserved for genderless Pok�mon.
 #define PERCENT_FEMALE(percent) min(254, ((percent * 255) / 100))
+#define OLD_UNOWN {							\
```

Comment:
i'd change this to `OLD_UNOWN_STAT_DATA` or something similar

Reply:
I'm actually no good at coming up with names. I did name it `OLD_UNOWN_DATA` previously.